docs/samples/Microsoft.ML.Samples/Dynamic/TextTransform.cs

```diff
@@ -15,7 +15,7 @@ public static void Example()
             var ml = new MLContext();
 
             // Get a small dataset as an IEnumerable and convert to IDataView.
-            var data = Microsoft.ML.SamplesUtils.DatasetUtils.GetSentimentData();
+            var data = GetSentimentData();
```

Comment:
>var [](start = 12, length = 3)

You have IEnumerable<SampleSentimentData> in the NGram sample. Let's adopt a consistent scheme, either var everywhere or the concrete type everywhere.

Reply:
Will do `var`.

---
In reply to: [278773485](https://github.com/dotnet/machinelearning/pull/3594#discussion_r278773485) [](ancestors = 278773485)